pom.xml

```diff
@@ -201,6 +202,28 @@
 			<version>3.0.0</version>
 			<scope>provided</scope>
 		</dependency>
+
+		<dependency>
+			<groupId>io.netty</groupId>
+			<artifactId>netty-codec-http</artifactId>
+			<version>4.1.0.Beta7</version>
```

Comment:
Latest netty stable release 4.0.33.FINAL is missing some features that are needed here. It might be possible to backport it from 4.1.0.Beta7 but as it is an experimental implementation of `DockerCmdExecFactory` I think its OK for now.